testFixtures/chuckThat.yml

```diff
@@ -0,0 +1,65 @@
+talonCommand: chuck that
+command:
+  actionName: delete
+  partialTargets:
+    - type: primitive
+      mark:
+        type: that
+  extraArgs: []
+languageId: python
+targets:
```

Comment:
I think I might call this `fullTargets` or `inferredFullTargets`? We might add a docstring to that field as well to indicate it's just for why-nots right now

Reply:
Added a comment and moved it to the end. Should be easy to remove if we decide it's not worth keeping.